docs/general-guidance/flight_identification.md

```diff
@@ -0,0 +1,120 @@
+# Flight Identification
+
+## GUFI
+
+### What is it?
+
+> The Global Unique Flight Identifier (GUFI) is intended to provide a unique reference to a specific flight, civil or military.
+> Its purpose is to assist in associating a message to the correct flight and to help in distinguishing between similar flights.
+
+> For the purpose of GUFI allocation “a flight” is considered to be the operation of an aircraft with a specified aircraft identification, at a specified departure aerodrome, at a specified date and time, from first submission of the flight plan (Preliminary or Filed) until in-blocks at an arrival aerodrome.
+
+> The operator, or its designated representative, is required to generate and allocate a GUFI to its FF-ICE flight plan.
+
+More information in chapter 3.7 GUFI & FLIGHT PLAN ASSOCIATION of the FF-ICE/R1 Implementation Guidance Manual.
+
+### Encoding a GUFI in FIXM Core 4.3.0
+
+`FIXM Core 4.3.0` supports the representation of the GUFI as specified in the draft d0.992 of the FF-ICE/R1 Implementation Guidance Manual. The GUFI consists of a version 4 UUID, as standardised by IETF RFC 4122 and ISO/IEC 9834-8:2014, supplemented with a namespace identifying the originator of the GUFI, and a creation timestamp. For further details, browse APPENDIX G – GUFI Construction of the FF-ICE/R1 Implementation Guidance Manual.
```

Comment:
In the top level introduction page of the manual, we refer to the FF-ICE IG as follows:
"ICAO FF-ICE Implementation Guidance Manual 0.99"
We should probably agree on an exact format for referring to it, and I'm fine with what is here. I we agree that this reference ("draft d0.992 of the FF-ICE/R1 Implementation Guidance Manual") I will change it in the top level page and anywhere else it appears.